src/main/resources/static/css/style.css

```diff
@@ -70,3 +70,69 @@ tr.solved {
 .table {
     --bs-table-bg: rgba(0, 0, 0, 0) !important;
 }
+
+/*toggle switch styling*/
```

Comment:
we are now combining 2 different things in a pr, your previous pr with the ui change and the the cypress tests, can we please separate them?

Reply:
I'm really sorry actually it was not intentional, I accidentally pushed that

Reply:
Can you remove all of it from this or or create a fresh one please?